src/OrchardCore.Modules/OrchardCore.Media.AmazonS3/AwsStorageOptionsConfiguration.cs

```diff
@@ -51,8 +52,7 @@ private void ParseBucketName(AwsStorageOptions options, TemplateContext template
             options.BucketName = template
                 .Render(templateContext, NullEncoder.Default)
                 .Replace("\r", String.Empty)
```

Comment:
Why we still replace `\r`?

Reply:
just in case some one saved config in CRLF mode and deployed it in unix based OS

Reply:
> just in case some one saved config in CRLF mode and deployed it in unix based OS

@neglectedvalue you're right, thanks for your patience.

Then @hishamco what happens if

> just in case some one saved config in LF mode and deployed it in windows based OS.

Your suggestion does the following on windows, which doesn't replace `\n` unix newlines.

    options.BucketName = template
        .Render(templateContext, NullEncoder.Default)
        .Replace("\r", String.Empty)
        .Replace(\r\n, String.Empty);

> We already used quite often https://github.com/OrchardCMS/OrchardCore/search?q=NewLine

As said, useful to generate strings but may be wrong to process strings from different sources.